sdk/resourcemanager/Azure.ResourceManager/src/Resources/Custom/Models/ResourceGroupExportResult.Serialization.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Text.Json;
+using Azure;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Resources.Models
+{
+    public partial class ResourceGroupExportResult : IOperationSourceProvider<ResourceGroupExportResult>
```

Comment:
We will need to only add this if net7 or higher if we end up going this route as this will fail to compile if a customer uses net6 and adds a dependency to our package